db/schema.rb

```diff
@@ -206,11 +206,20 @@
     t.datetime "deleted_at"
   end
 
+  create_table "collections_device_descriptions", force: :cascade do |t|
```

Comment:
haben wir das model in diesem Branch schon? Ggf aus dem schema des anderen Branches?

Reply:
Hab ich die eine Stelle übersehen.
Der baucht die schema.rb immer nach meiner lokalen db und nicht nach den migrations